docker/cassandra/install-cassandra

```diff
@@ -1,7 +1,27 @@
 #!/bin/bash -xue
 
+echoerr() { >&2 echo $@; }
+
 cassandra_version=$1
 
+case $cassandra_version in
+3.*)
+    # supported
+    ;;
+
+2.*)
```

Comment:
we should probably just remove this one

Reply:
Put this here as more of a "here be dragons" to me or anyone else in the future.